src/main/java/org/ibp/api/brapi/v2/list/GermplasmListResourceBrapi.java

```diff
@@ -0,0 +1,102 @@
+package org.ibp.api.brapi.v2.list;
+
+import com.fasterxml.jackson.annotation.JsonView;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import org.apache.commons.lang3.StringUtils;
+import org.generationcp.middleware.domain.search_request.brapi.v2.GermplasmListSearchRequestDTO;
+import org.generationcp.middleware.service.api.BrapiView;
+import org.generationcp.middleware.service.api.GermplasmListDTO;
+import org.ibp.api.brapi.GermplasmListServiceBrapi;
+import org.ibp.api.brapi.v1.common.BrapiPagedResult;
+import org.ibp.api.brapi.v1.common.EntityListResponse;
+import org.ibp.api.brapi.v1.common.Metadata;
+import org.ibp.api.brapi.v1.common.Pagination;
+import org.ibp.api.brapi.v1.common.Result;
+import org.ibp.api.domain.common.PagedResult;
+import org.ibp.api.rest.common.PaginatedSearch;
+import org.ibp.api.rest.common.SearchSpec;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Api(value = "BrAPI v2 Germplasm List Services")
+@Controller(value = "GermplasmListResourceBrapi")
+public class GermplasmListResourceBrapi {
```

Comment:
Rename resource as: `ListResourceBrapi` to conform to Brapi spec (albeit we only support germplasm list types). Change @Api value as well